lib/location/location_core.c

```diff
@@ -647,7 +647,7 @@ static void location_core_method_timeout_work_fn(struct k_work *work)
 
 	ARG_UNUSED(work);
 
-	LOG_WRN("Method specific timeout expired");
+	LOG_DBG("Method specific timeout expired");
```

Comment:
I need these traces as a default from MoSh output and on the other hand, in DBG level there comes too much stuff. 

So, instead of this I'd propose:
change these traces to INF level in location lib AND change ATv2 config for loc lib to have: CONFIG_LOCATION_LOG_LEVEL_WRN=y

From ATv2 output point of view with a current usage, the result would be the same.

Reply:
The default log level has been causing problems already earlier. IMHO it would be better if the default log level would be WRN, then there would be two possible steps for increasing logging when needed instead of just one.

Changing the Location library log level in ATv2 prj.conf instead is fine for me. Any opinions, @simensrostad and @trantanen?

Reply:
Thanks!